backend/sensorStore.js

```diff
@@ -0,0 +1,49 @@
+// Similar to voteStore
+//const server = require('/server')
+const fs = require('fs');
+
+class sensorStore {
+    constructor() {
+        this.keys = new Array
+        this.length = 0;
+        this.defaultValue = null;
+    }
+
+    populate(){
+        let rawdata = fs.readFileSync('mockSensorData.json');
+        let sensors = JSON.parse(rawdata).sensors;
+        this.keys = sensors
+    }
+    
+    store(id, location, temperature, time) {
+        if (isFinite(id)) {     // had issues with "instanceof"
```

Comment:
I think checking keys by ID wont work, should probably do a for loop to iterate through and find the associated item.